uber/site_sections/preregistration.py

```diff
@@ -438,7 +438,10 @@ def transfer_badge(self, session, message='', **params):
                 if attendee.full_name in c.BANNED_ATTENDEES:
                     send_banned_email(attendee)
 
```

Comment:
This is a semi-related fix.  When transferring a badge, if someone decides to kick in extra, we now immediately redirect them to the Attendee Donation form.